pydantic_ai_slim/pydantic_ai/ag_ui.py

```diff
@@ -128,6 +132,7 @@ def __init__(
         usage: Usage | None = None,
         infer_name: bool = True,
         toolsets: Sequence[AbstractToolset[AgentDepsT]] | None = None,
+        on_complete: AgentRunCallback | None = None,
```

Comment:
Please remove this from `AGUIApp` and `Agent.to_ag_ui` as those are really meant to only be used for cases where you want to serve the agent quickly and not do any pre or post processing.